docker/backend.dockerfile

```diff
@@ -67,13 +67,13 @@ COPY ./docker/plugins.py /app/plugins.py
 ARG PLUGIN_CSV
 # ="enabled,name,git_url,git_branch\n"
 ENV PLUGIN_INSTALL_PATH /app/plugins
-RUN python plugins.py \
+RUN python3 plugins.py \
    --plugin_csv ${PLUGIN_CSV} \
    --install_path ${PLUGIN_INSTALL_PATH} \
    --method download
 
 # Run before_vips() from plugins prerequisites
-RUN python plugins.py \
+RUN python3 plugins.py \
```

Comment:
Probably a consequence of stacking commits on this PR.
Can you update it so that it matches the Dockerfile on ULiege repository ? 
To avoid issues with "python" and "python3", I've added https://github.com/Cytomine-ULiege/pims/blob/859b175621372683c8ce3d72761d6bc37179cd05/docker/backend.dockerfile#L40
and then I can always use "python".